mmv1/products/compute/RegionSecurityPolicyRule.yaml

```diff
@@ -0,0 +1,216 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'RegionSecurityPolicyRule'
+min_version: beta
+base_url: 'projects/{{project}}/regions/{{region}}/securityPolicies/{{security_policy}}'
+self_link: 'projects/{{project}}/regions/{{region}}/securityPolicies/{{security_policy}}/getRule?priority={{priority}}'
+create_url: 'projects/{{project}}/regions/{{region}}/securityPolicies/{{security_policy}}/addRule?priority={{priority}}'
+update_verb: :POST
+update_mask: true
+update_url: 'projects/{{project}}/regions/{{region}}/securityPolicies/{{security_policy}}/patchRule?priority={{priority}}'
+delete_verb: :POST
+delete_url: 'projects/{{project}}/regions/{{region}}/securityPolicies/{{security_policy}}/removeRule?priority={{priority}}'
+description: |
+  A rule for the RegionSecurityPolicy.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Creating region security policy rules': 'https://cloud.google.com/armor/docs/configure-security-policies'
+  api:
+    'https://cloud.google.com/compute/docs/reference/rest/beta/regionSecurityPolicies/addRule'
+id_format: 'projects/{{project}}/regions/{{region}}/securityPolicies/{{security_policy}}/priority/{{priority}}'
+import_format: ['projects/{{project}}/regions/{{region}}/securityPolicies/{{security_policy}}/priority/{{priority}}']
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    kind: 'compute#operation'
+    path: 'name'
+    base_url: 'projects/{{project}}/regions/{{region}}/operations/{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'targetLink'
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'status'
+    complete: 'DONE'
+    allowed:
+      - 'PENDING'
+      - 'RUNNING'
+      - 'DONE'
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error/errors'
+    message: 'message'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'region_security_policy_rule_basic'
+    primary_resource_id: 'policy_rule'
+    min_version: 'beta'
+    vars:
+      sec_policy_name: 'policyruletest'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'region_security_policy_rule_multiple_rules'
+    primary_resource_id: 'policy_rule_one'
+    min_version: 'beta'
+    vars:
+      sec_policy_name: 'policywithmultiplerules'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'region_security_policy_rule_with_network_match'
+    # it needs to run synchronously because a region can have only one google_compute_network_edge_security_service.
+    # there is a robust handwritten test which covers this scenario. 
+    skip_test: true
+    primary_resource_id: 'policy_rule_network_match'
+    min_version: 'beta'
+    vars:
+      sec_policy_name: 'policyfornetworkmatch'
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'region'
+    description: |
```

Comment:
Should add `immutable: true` to the `region` field? The use case is that when the region changes, should it create a new resource?

Reply:
Done